testsuite/MDAnalysisTests/data/lammps/albite_triclinic.dump

```diff
@@ -0,0 +1,52 @@
+ITEM: TIMESTEP
```

Comment:
As far as I can tell this has two identical timesteps, I am happy if you truncate it to one.